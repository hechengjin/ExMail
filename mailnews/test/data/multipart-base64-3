Content-Type: multipart/mixed; boundary=boundary

etc etc
--boundary
Content-Type: text/html
Content-Transfer-Encoding: base64

PGh0bWw+PGhlYWQ+VGhpcyB0aW1lLCB0aGUgdGFncw0Kc2hvdWxkIGJlIHN0cmlwcGVkIG91dC48L2hlYWQ+PC9odG1sPg==

--boundary--
