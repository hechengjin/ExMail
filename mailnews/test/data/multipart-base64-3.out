This time, the tags should be stripped out.